### PR TITLE
Fix line break in card url

### PR DESCRIPTION
**Pull request by Morphclue**
# Context 
Currently the line break isn't correct for medium devices (#161).
With this change it will be fixed:
image

### CODE REVIEW

File: apps/frontend/src/app/dashboard/card/card.component.html [hunk 1]
```diff
@@ -14,7 +14,7 @@ <h5 class="card-title">{{ poll.title }}</h5>
       </div>
     </div>
   </div>
-  <div class="card-body">
+  <div class="card-body p-2">
```
Comment (Clashsoft) [inline apps/frontend/src/app/dashboard/card/card.component.html:17]: Why this change?

Comment (Morphclue) [reply apps/frontend/src/app/dashboard/card/card.component.html:17]: Cause otherwise something like this might happen on small devices:
image

Comment (Clashsoft) [reply apps/frontend/src/app/dashboard/card/card.component.html:17]: But the smaller padding looks a little off imo
